### ISSUE TITLE
[Request] rxjs-prefer-angular-composition for services.

### ISSUE THREAD

**Issue by distante** [cartant/rxjs-tslint-rules#120]
Angular services also can use the `OnDestroy`, if subscriptions are done there for background tasks, we should be able to link them as "use a subscription object and unsubscribe on `OnDestroy`"

**Comment by cartant** [comment]
The ESLint version of this rule supports classes other than components. I'm not really interested in adding features/functionality to this package of rules now that TSLint has reached its end of life.

**Comment by distante** [comment]
Fair enough. I can not change my project to ESLint at the time until all my dependencies support it so I supposse I have to just live with it.
